kt-android-runtime/app/src/main/res/drawable/ic_launcher_background.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Remove this file

Reply:
I'm thinking of having an example app on this module. These files will be used then

Reply:
I'm resolving the duplicates of this comment.

Reply:
what do you think about having an example app, @lbguilherme ?